### DISCUSSION TITLE
Import ip addresses through csv prepared in excel

### DISCUSSION THREAD

**Discussion by fractal90** [netbox-community/netbox#14836] [Help Wanted!]
Hi, Why when I try to import ip addresses from csv I get the error?

<class 'UnicodeDecodeError'>

'utf-8' codec can't decode byte 0xcc in position 579: invalid continuation byte

Python version: 3.10.12
NetBox version: 3.6.6
Plugins: None installed

**Comment by candlerb** [comment]
I'd guess you're trying to send a CSV file in a non-unicode character set (e.g. Windows-1252) when Netbox is expecting unicode. The error message tells you the byte offset (579) where the error occurs.

You could use a utility such as iconv to convert the character encodings. Or you might be able to tell Excel to export in unicode.

**Reply by fractal90** [reply]
I tried to save csv file with encoding unicode utf-8 (Excel 2019) but it didn't work. I had to open csv in the notepad++ and copy body to new file in the notepad ++ and after it I could import
